ddtrace/internal/datastreams/processor.py

```diff
@@ -350,8 +355,8 @@ def set_checkpoint(self, tags, now_sec=None, payload_size=0, span=None):
         if "direction:out" in tags:
             # Add the header for this now, as the callee doesn't have access
             # when producing
-            payload_size += len(ctx.encode())
-            payload_size += len(PROPAGATION_KEY)
+            payload_size += len(ctx.encode_b64())
+            payload_size += len(PROPAGATION_KEY_BASE_64)
```

Comment:
greatest nit of all time
```suggestion
            payload_size += len(ctx.encode_b64()) + len(PROPAGATION_KEY_BASE_64)
```